util/generate_robot_docs.js

```diff
@@ -164,5 +164,5 @@ const page = [
     "Autogeneration timestamp: " + new Date().toISOString()
 ]
 
-fs.writeFileSync(path.join(__dirname, "../docs/_pages/general/supported-robots.md"), page.join("\n"))
+fs.writeFileSync(path.join(__dirname, "../docs/_pages/general/supported-robots.md"), page.join("\n") + "\n")
```

Comment:
That also applies to markdown?

Reply:
I consider Markdown to be text, not binary.